src/ws-dropdown-item.js

```diff
@@ -0,0 +1,126 @@
+import {React, Component} from './imports';
+import {WSDropdownMenu} from './ws-dropdown-menu';
+
+export class WSDropdownItem extends Component {
+
+  /**
+   * @type {Object}
+   */
+  static defaultProps = {
+    item: null,
+    icon: null,
+    multiple: false,
+    isParent: false,
+    handle: () => {}
+  };
+
+  /**
+   * @type {Object}
+   */
+  static propTypes = {
+    item: React.PropTypes.object,
+    icon: React.PropTypes.string,
+    multiple: React.PropTypes.bool,
+    isParent: React.PropTypes.bool,
+    handle: React.PropTypes.func
+  };
+
+  /**
+   * @type {Object}
+   */
+  static contextTypes = {
+    multiple: React.PropTypes.bool
+  };
+
+  /**
+   * @param props {Object}
+   * @param context {Object}
+   * @constructor
+   */
+  constructor(props, context) {
+    super(props, context);
+    this.state = props.item;
+    this.menu = null;
+  }
+
+  /**
+   * Important to update the state when the items in menu changed.
+   * When in the multi select a dropdown item is selected and stored (submitted) it will be filtered out
+   * of the item list and shown in a separate value list. React doesn't created new components but reuses them.
+   * This leads to property updates and without this function a wrong item will be rendered.
+   * @param props {Object}
+   * @void
+   */
+  componentWillReceiveProps(props) {
+    this.state = props.item;
+  }
+
+  /**
+   * Handle clicks on this dropdown item. This can trigger a back navigation, selecting the item on multi selects
+   * or change the dropdown value for simple dropdown's
+   * @param event {MouseEvent}
+   * @void
+   */
+  onClick(event) {
+    event.stopPropagation();
+    // Click on parent means back navigation
+    if (this.props.isParent) {
+      // dropdown-item(go-back) -> dropdown-menu(show-parent) -> dropdown-item(show-parent) -> dropdown-menu
+      this.props.handle('go-back');
```

Comment:
Though about that too but decided against it since this handle function covers up to 6 cases I don't want to add 6 callbacks to all dropdown items/menus. 
https://github.com/wholesale-design-system/components/pull/8/files/efef757429e3a37f0465dac9a46714a66c02ad47#diff-2a426cf03d9e6062ec7e0e3c2862ceeaR133

Also I wanted a consistent behaviour through all dropdown components.